group_membership.tf

```diff
@@ -0,0 +1,22 @@
+# Put database users in the appropriate group.
+resource "aws_iam_user_group_membership" "read_only_users" {
+  provider = aws.users
+  for_each = toset([for username, attributes in var.users : username if attributes["role"] == "read_only"])
```

Comment:
Alphabetize.

```suggestion
  for_each = toset([for username, attributes in var.users : username if attributes["role"] == "read_only"])
  provider = aws.users
```

Reply:
We typically put the provider first because of its importance, so I'm hesitant to alphabetize in this particular case.  @jsf9k - can you be our tie-breaker here?

Reply:
@mcdonnnj, are you ok resolving this without alphabetizing here?

Reply:
That's fine for now but I'd like to have a conversation about this. We have very inconsistent styling because nothing is firmly enforced with our linting suite.

Reply:
Sounds good, thanks.